test/workspace.js

```diff
@@ -249,4 +249,61 @@ describe('Workspace', function() {
       });
     });
   });
+
+  describe('Creating 2 projects from template', function() {
+    beforeEach(givenEmptySandbox);
+    afterEach(function setWorkspaceToSandboxDir() {
+      process.env.WORKSPACE_DIR = SANDBOX;
+    });
+
+    it('Create note app & hello world app in same directory and switching workspace',
```

Comment:
it('creates `noteapp` & `helloworldapp` in the same dir while switching workspaces', ...